### PR TITLE
Adding support for CRX-10iA/L

### PR DESCRIPTION
**Pull request by demorise**
Adding support for the crx10iA/L cobot. I realized someone tried to do this in the past but couldn't get through with it due to formatting issues. I have tried as much as possible to be consistent with other support packages in the repository. We have used the URDF succesfully with our cobot and the R30iB mini plus controller without any modifications to the fanuc ROS driver.

@gavanderhoorn 

crx10_collision_model

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Thanks for the PR. It's certainly appreciated.

I've been rather busy with other things. I'll take a look at a later time.

**Comment by gavanderhoorn** [comment]
I've pushed a few fixup commits and submitted a PR @demorise.

See https://github.com/WILDER-SYSTEMS-LLC/fanuc_experimental/pull/1.

**Comment by demorise** [comment]
> I've pushed a few fixup commits and submitted a PR @demorise.
> 
> See [WILDER-SYSTEMS-LLC#1](https://github.com/WILDER-SYSTEMS-LLC/fanuc_experimental/pull/1).

Thanks @gavanderhoorn . Is there any action I need to take from my end?

**Comment by gavanderhoorn** [comment]
If you agree with my proposed changes, you could merge the PR.

File: fanuc_experimental/package.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
   <exec_depend>fanuc_r2000ib_support</exec_depend>
   <exec_depend>fanuc_r2000ic_support</exec_depend>
+  <exec_depend>fanuc_crx10ia_support</exec_depend>
```
Comment (gavanderhoorn) [inline fanuc_experimental/package.xml]: Missed this one: could you please keep the list alphabetically sorted?

File: fanuc_crx10ia_support/package.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <p>
+      ROS-Industrial support for the Fanuc CRX-10iA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Fanuc CRX-10iA manipulators. This currently includes the /L model only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>CRX-10iA/L - 1418mm reach</li>
```
Comment (gavanderhoorn) [inline fanuc_crx10ia_support/package.xml:16]: Normally we list whether the model was created based on the regular joint limits (for J1 fi) or the extended ones, or whether a hollow-wrist config was used or not.

I don't believe the CRX-10iA/L has those kind of configuration options, so this is OK for now.

File: fanuc_crx10ia_support/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# fanuc_crx10ia_support
+
+## Overview
+
+This package is part of the [ROS-Industrial][] program. See the main
+[fanuc][] page on the ROS wiki for more information on usage.
```
Comment (gavanderhoorn) [inline fanuc_crx10ia_support/readme.md]:
```suggestion
[fanuc_experimental][] page on the ROS wiki for more information on usage.
```

File: fanuc_crx10ia_support/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+## Contents
+
+See `package.xml` for information about the contents of this package.
+
+
+[ROS-Industrial]: http://wiki.ros.org/Industrial
+[fanuc]: http://wiki.ros.org/fanuc
```
Comment (gavanderhoorn) [inline fanuc_crx10ia_support/readme.md]:
```suggestion
[fanuc_experimental]: http://wiki.ros.org/fanuc_experimental
```

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
Some additional comments.

File: fanuc_crx10ia_support/package.xml (hunk 2 above)
Comment (gavanderhoorn) [reply fanuc_crx10ia_support/package.xml:16]: This is not an action for you. Just wanted to add a note here.

**Comment by demorise** [comment]
I have made the changes and pushed a new commit.

**Comment by gavanderhoorn** [comment]
For your next PR: Github makes accepting suggestions really easy. Just click the button(s) in the comments in the web ui.

**Comment by gavanderhoorn** [comment]
Apparently Actions needed a nudge.

**Comment by demorise** [comment]
> For your next PR: Github makes accepting suggestions really easy. Just click the button(s) in the comments in the web ui.

Thanks!

**Review by gavanderhoorn — APPROVED** [review]
Alright, this looks OK now.

Thanks again for the PR @demorise, and thanks for iterating on it with me.

:+1:

**Comment by gavanderhoorn** [comment]
I'll use a squash-merge, as we don't need all the fixups to end up in the repository's commit history.

**Comment by gavanderhoorn** [comment]
Thanks again @demorise.

Note that because I used a squash-merge, Github may not let you know you can delete the branch.

**Comment by demorise** [comment]
> Thanks again @demorise.
> 
> Note that because I used a squash-merge, Github may not let you know you can delete the branch.

No worries!

**Pull request merged**